spec/Drivetrain/BatteryManagement.vspec

```diff
@@ -10,25 +10,47 @@
 # Battery management system for EV / Hybrids
 #
 
-- BatteryTemperature:
+- Battery:
+  type: branch
+  description: Battery status
+
+- Battery.Temperature:
   datatype: float
   type: sensor
   unit: celsius
   description: Temperature of the battery pack
 
-- BatteryCapacity:
-  datatype: int8
+- Battery.StateOfCharge:
+  datatype: uint8
   type: sensor
   unit: percent
   min: 0
   max: 100
-  description: Remaining capacity of the batter pack
+  description: The current charging level as a percentage of original battery performance.
 
-- LowBatteryLevel:
+- Battery.LowLevel:
   datatype: Boolean
   type: sensor
   description: Indicates that the battery level is low
 
+- Battery.AccumulatedChargeEnergy:
+  datatype: float
+  type: sensor
+  unit: kWh
+  description: The accumulated energy delivered to the battery during charging.
+
+- Battery.AccumulatedConsumedEnergy:
+  datatype: float
+  type: sensor
+  unit: kWh
+  description: The accumulated energy leaving HV battery for propulsion and auxiliary loads.
```

Comment:
Since when?  How and when is this signal reset to zero?

Reply:
I believe this could be used to estimate how "worn out" a battery is likely to be.  (together with AccumulatedChargeEnergy).  It is similar to odometer, the "complete distance travelled" for evaluating how worn a vehicle is, but more accurate for the battery itself.

Reply:
I would conclude this one.

Reply:
I still think this leaf is statistics relevant. So I think it is better to introduce following leaf: Battery.Statistics.AccumulatedConsumedEnergy

I also think we need tree structure which is generic and can be instantiated for the life time, from last charging or from date x, 

Please also add a description seans it is for the lifetime of the vehicle.